src/connector/postgres/conversion.rs

```diff
@@ -283,6 +300,10 @@ impl<'a> ToSql for ParameterizedValue<'a> {
                 }
                 _ => float.to_sql(ty, out),
             },
+            ParameterizedValue::Enum(string) => {
+                out.extend_from_slice(format!("{}", string).as_bytes());
```

Comment:
Why you double allocate with `format`, don't you have the ownership to the string, so you can just call `as_bytes` directly?

Reply:
fixed